rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/executor/BakePoller.groovy

```diff
@@ -120,6 +125,11 @@ class BakePoller implements ApplicationListener<ContextRefreshedEvent> {
                             if (!cancellationSucceeded) {
                               bakeStore.removeFromIncompletes(roscoInstanceId, statusId)
                             }
+
+                            // This will have the most up-to-date timestamp.
```

Comment:
This path specifically handles the case where the polling agent detects incomplete bakes that are not managed by _this_ rosco instance, and have not been updated beyond some threshold. If this instance is updating the bake, we will not end up here.